### ISSUE TITLE
Version 0.11.3 issue fetching data

### ISSUE THREAD

**Issue by thamudi** [johannschopplich/nuxt-api-party#24]
### Environment

- Operating System: Linux
- Node Version:     v18.14.0
- Nuxt Version:     3.5.2
- Nitro Version:    2.4.1
- Package Manager:  yarn@1.22.19
- Builder:          vite
- User Config:      app, modules, css, i18n, runtimeConfig, typescript
- Runtime Modules:  nuxt-api-party@0.10.5, @nuxtjs/html-validator@1.3.1, @nuxtjs/i18n@8.0.0-beta.12, @vueuse/nuxt@10.1.2, @nuxtjs/tailwindcss@6.7.0, @nuxt/image-edge@1.0.0-28089462.3520a62
- Build Modules:    -

### Reproduction

Here's a [link](https://stackblitz.com/edit/github-8hcqp4?file=app.vue) to StackBlitz with a minimal reproduction

Note that the reproduction was recreated using the [Getting started](https://nuxt-api-party.byjohann.dev/guide/getting-started.html) steps.

### Describe the bug

On ver 0.11.3 I cant seem to get any data. 
When viewing the ref in vue dev tools I get the following error
This screen shot was taken from my project
![Screenshot 2023-05-29 163820](https://github.com/johannschopplich/nuxt-api-party/assets/18720062/d7137577-7ea6-4828-949f-12ba8013ccff)

### Additional context

Initially I was trying to fetch data using `useAsyncData` 
Here's a snippet of the code I was trying to work with.

```js
<script setup>
  import qs from 'qs'
  import { get } from '@vueuse/core'
  const query = computed(() =>
    qs.stringify(
      {
        populate: 'deep',
      },
      {
        encodeValuesOnly: true,
      }
    )
  )

  const { data: footerSeo, error } = await useAsyncData(
    'footer',
    () => {
      return $cms(`seo?${get(query)}`, { cache: true })
    },
    {
      watch: [query],
    }
  )

  const footerData = computed(
    () => get(footerSeo).data.attributes.seo.metaSocial
  )
</script>
```

I want to note that downgrading the package to v `0.10.5` fixed this issue and the above code works.

### Logs

```shell-script
statusCode: 500                                                                                                 
data: undefined                                                                                                 
statusMessage: undefined
```

**Comment by thamudi** [comment]
Hey @DeVoresyah 
No, like I mentioned I am using [useAsyncData](https://nuxt.com/docs/api/composables/use-async-data)
Nuxt 3 provides its own solution for fetching data that uses [ofetch](https://nuxt.com/docs/getting-started/data-fetching#fetch) under the hood. That is why I don't use axios in my nuxt3 projects.

**Comment by johannschopplich** [comment]
@thamudi Thanks for the reproduction!
Quick question. Why are you not using [`useCmsData`](https://nuxt-api-party.byjohann.dev/api/use-api-party-data.html)? It's already a wrapper around `useAsyncData` and should be used preferrabley.

Will take a look at the error later.

**Comment by thamudi** [comment]
@johannschopplich 
I didn't know it was a wrapper for the useAsyncData, thanks for the reference!

**Comment by arif-rh** [comment]
I have the same issue and downgrading into 0.10.5 as @thamudi  mention is working.

**Comment by johannschopplich** [comment]
@thamudi @arif-rh I've found the issue. You seem to use the `runtimeConfig` to define API endpoints, instead of using the `apiParty` module options. The first one was not resolved correctly. 

Fix is available in v0.11.4.

**Comment by thamudi** [comment]
Just tested it. 
Thanks for the follow up @johannschopplich
